xbmc/interfaces/json-rpc/FileItemHandler.cpp

```diff
@@ -77,6 +77,10 @@ bool CFileItemHandler::GetField(const std::string &field, const CVariant &info,
         result[field] = StringUtils::Split(info[field].asString(), EPG_STRING_TOKEN_SEPARATOR);
         return true;
       }
+      else if (field == "uniqueid" && result["channelid"].isNull())
+	  {
```

Comment:
Indentation of brackets is incorrect. Can we also add a comment here detailing why this needs to be here? Otherwise no one will know the purpose.

Reply:
Ha, @phunkyfish you beat me to this. ;-)